packages/view-types/spatial/src/SpatialOptions.js

```diff
@@ -83,25 +83,25 @@ export default function SpatialOptions(props) {
 
   return (
     <OptionsContainer>
-      {canShowColorEncodingOption ? (
+      {canShowColorEncodingOption && (
```

Comment:
Can we revert this and the other analogous changes? It does not seem related to accessibility https://github.com/vitessce/vitessce/blob/main/dev-docs/design-guidelines.md#jsx

Reply:
Done in commit 4e4f4a55